askama_hyper/tests/basic.rs

```diff
@@ -1,6 +1,6 @@
 use std::convert::Infallible;
 
-use askama::Template;
+use askama_hyper::Template;
```

Comment:
Why changing this (same for all others)?

Reply:
This seems to be a misunderstanding of "unused imports", should I revert it?